src/main/realtime.ts

```diff
@@ -12,42 +12,23 @@ function cleanAndStartRemoteNotifications() {
   stopRemoteNotifications();
 
   socket = io(process.env.NOTIFICATIONS_URL, {
+    transports: ['websocket', 'polling'],
     auth: {
       token: obtainToken('bearerToken'),
     },
     withCredentials: true,
   });
 
-  socket.io.on('open', () => {
-    socket?.io.engine.transport.on('pollComplete', () => {
```

Comment:
Setting the cookie is required for the load balancer of any provider to be able to connect always against the same replica which holds the websocket

Reply:
I believe this was for sticky sessions, but the polling transport was not connecting either, if we use the websocket transport, which is working, the connection is always open right?

Reply:
I restored the ingress cookie